### PR TITLE
Misc. improvements in the Console component introduction

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: components/console/introduction.rst [hunk 1]
```diff
@@ -140,6 +140,9 @@ output. For example::
     // white text on a red background
     $output->writeln('<error>foo</error>');
 
+The closing tag can be replaced by just ``</>``. This makes the component to
+stop applying any style defined by the immediately previous open tag.
```
Comment (xabbuh) [inline components/console/introduction.rst]: I would reword this a bit:

> The closing tag can be replaced by ``</>`` which simply closes the last opened tag.

Comment (javiereguiluz) [inline components/console/introduction.rst]: I don't like my original text ... but I don't like either the new proposal. My fear is that reader would say: *obviously* the close tag closes the opened tag!!!  In this case, the close tag is not about closing tags but about "neutralizing" the effects applied in the previously opened tag. I don't know if I explained myself because it's not easy to explain what I want to say :(

Comment (xabbuh) [inline components/console/introduction.rst]: Maybe then something like

> The closing tag can be replaced by `</>` which revokes all formatting options established by the last opened tag.

Comment (javiereguiluz) [inline components/console/introduction.rst]: I like it. I've just updated it. Thanks!

**Comment by xabbuh** [comment]
:+1:

File: components/console/introduction.rst [hunk 2]
```diff
@@ -148,23 +151,26 @@ It is possible to define your own styles using the class
...
+    $output->writeln('<fire>foo</>');
 
 Available foreground and background colors are: ``black``, ``red``, ``green``,
 ``yellow``, ``blue``, ``magenta``, ``cyan`` and ``white``.
 
-And available options are: ``bold``, ``underscore``, ``blink``, ``reverse`` and ``conceal``.
+And available options are: ``bold``, ``underscore``, ``blink``, ``reverse``
+(enables the "reverse video" mode where the background and foreground colors
+are swapped) and ``conceal`` (the text will be hidden; commonly used to ask
+the user to type sensitive information).
```
Comment (WouterJ) [inline components/console/introduction.rst]: I don't think this is a very good explanation of conceal. It just makes the foreground color transparent. The text is still there (and visible when copy/pasting the terminal text). Besides that, the "input box" of questions can't be styled.

File: components/console/introduction.rst [hunk 3]
```diff
@@ -140,6 +140,9 @@ output. For example::
     // white text on a red background
     $output->writeln('<error>foo</error>');
 
+The closing tag can be replaced by ``</>`` which revokes all formatting options
```
Comment (WouterJ) [inline components/console/introduction.rst]: "[...], which [...]"

File: components/console/introduction.rst (hunk 2 above)
Comment (javiereguiluz) [inline components/console/introduction.rst]: I've reworded the explanation. Thanks.

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
Thanks Javier.
